app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -277,8 +277,8 @@ public function create_orderAction() {
             $computedCart = Mage::getModel('boltpay/boltOrder')->buildCart($immutableQuote, false );
             $this->boltHelper()->notifyException($orderCreationException, array( 'magento_order_details' => json_encode($computedCart)));
             //////////////////////////////////////////////////////
-
-        } finally {
+            // finally requre PHP => 5.5
+//         } finally {
```

Comment:
change finally to catch and call `benchmark( "Response sent to Bolt")` before catch so we can keep logic same

Reply:
@daisy1754, done

Reply:
let's just this line `//         } finally {`